draft-ietf-quic-manageability.md

```diff
@@ -696,7 +696,11 @@ requirements.
 The QUIC header optionally contains a Connection ID which can be used as
 additional entropy beyond the 5-tuple, if needed. The QUIC handshake needs
 to be observed in order to understand whether the Connection ID is present and
-what length it has.
+what length it has. However, Connection IDs may be renegotiated during
```

Comment:
Perhaps it should be explicitly mentioned that even the length of the CIDs can change. A change in length could break any logic that observes a handshake and assumes all CIDs have the same length.

Not to mention that the tuple itself could be shared by several parallel connections, each with different CIDs (possibly with different lengths).